Examples/Pass/arc.cvf

```diff
@@ -44,16 +34,12 @@ method drop() {
       {| noCnt() |}
         <free = (true)>;
       {| emp |}
-    } else {
-      {| emp |}
-        dummy = dummy; // skip;
-      {| emp |}
     }
   {| emp |}
 }
 
 view error();
```

Comment:
That's actually in `mw-pldi-experimental`.  I was considering cherry-picking it into this PRQ but it didn't merge cleanly so I gave up.  I could try pulling it across and back porting it if you want.